components/cmp/modal/src/index.scss

```diff
@@ -0,0 +1,158 @@
+@import '~@schibstedspain/sui-theme/lib/index';
+@import '~@schibstedspain/sui-atom-button/lib/index';
+@import '~@s-ui/react-atom-switch/lib/index';
+
+$bgc-sui-cmp-modal: rgba(33, 41, 52, .85) !default;
+
+.sui-CmpModal {
+  align-items: center;
+  background: $bgc-sui-cmp-modal;
+  bottom: 0;
+  color: $c-gray-dark;
+  display: flex;
+  justify-content: center;
+  left: 0;
+  line-height: $lh-m;
+  overflow-y: scroll;
+  position: fixed;
+  right: 0;
+  top: 0;
+  transition: opacity .15s ease;
+  z-index: 2147483647;
```

Comment:
!?

Reply:
hahaha I used it for testing but, tbh, it's the maximum value of the z-index. As this should be above everything and avoid any kind of ad or navigation. :) Any idea is welcome, but I've to be sure this is over everything.

Reply:
No better idea. We lack of a layer convention...
Let's keep it. It'll be easily changeable.